provision/kubernetes/kubernetes.tf

```diff
@@ -95,11 +95,11 @@ resource "openstack_compute_floatingip_associate_v2" "floatip_1" {
   }
 
   provisioner "local-exec" {
-    command = "./prepare_template ${openstack_compute_instance_v2.basic.network.0.fixed_ip_v4} ${var.routerip}"
+    command = "../../${var.main_directory_name}/${var.user_name}/${var.machine_name}/prepare_template ${openstack_compute_instance_v2.basic.network.0.fixed_ip_v4} ${var.routerip} ${var.main_directory_name} ${var.user_name} ${var.machine_name}"
```

Comment:
This is pretty fragile. If I touch this path, multiple places may break. In fact, this whole command is pretty scary.

Reply:
Done.